pcl_conversions/include/pcl_conversions/pcl_conversions.h

```diff
@@ -717,191 +723,23 @@ namespace pcl {
 
 } // namespace pcl
 
-namespace ros
+namespace message_filters
 {
   template<>
   struct DefaultMessageCreator<pcl::PCLPointCloud2>
   {
-    boost::shared_ptr<pcl::PCLPointCloud2> operator() ()
+    std::shared_ptr<pcl::PCLPointCloud2> operator() ()
     {
-      boost::shared_ptr<pcl::PCLPointCloud2> msg(new pcl::PCLPointCloud2());
+      std::shared_ptr<pcl::PCLPointCloud2> msg(new pcl::PCLPointCloud2());
       return msg;
     }
   };
-
+  
+  //https://github.com/ros2/message_filters/commit/46e1229a1d8c0ecca68e01b9cf0d8c13f9f6f87a#diff-651c2688431bf33a7ed4f0c68f9d86d2
   namespace message_traits
   {
-    template<>
```

Comment:
Sorry maybe I was too non-specific, can you add ALL of this stuff back in, but then just comment it ALL out since there is no type masquerading yet? Revert to `namespace ros {...}` and just block comment it all out with a `TODO` on type masquerading.